packages-native/progress-bar/src/__tests__/ProgressBar.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe("ProgressBar", () => {
 
     it("renders no progress and an error when the value is greater than the maximum", () => {
         const component = render(<ProgressBar {...createProps(150, 0, 100)} />);
-        expect(component.getByType(Bar).props.progress).toBe(0);
+        expect(component.getByType(Bar).props.progress).toBe(1);
         expect(component.getByType(Text).props.children).toBe(
```

Comment:
We shouldn't expect any errors anymore ?

Reply:
We should, it will fit 100% and show the error